releases/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore everything in this directory
```

Comment:
We could also just add `/releases/*` to the top-level .gitignore to have all the ignoring info in one place, but not a blocker.